### PR TITLE
Bottom nav bar

### PR DESCRIPTION
**Pull request by byeongjae-kang**
## Description

Please describe and/or use bullet points to to explain what this PR is about.
 - fix circular indepency while importing - Typography warning message should be gone now
 - create screens to navigate to right screens
 - create icons based on what designers have created
 - add styles to bottom navigation
 - add story for visual testing
 - add unit and snapshot tests
 - update script not to build each time to run application
 - ignore android foler

## Jira Ticket

Please copy and paste Jira issue link here.
https://fridge-manager.atlassian.net/browse/IRM-181

## Images or Videos for review

Please try to add images or videos that are helpful to understand what this PR is about.

https://github.com/byeongjae-kang/ate-fe/assets/82554049/ab5d8eb8-ec1c-4426-9b53-f463278f6d0a

## Check List

Please add `x` in the list below where applicable.

- [x] Test added or updated?
- [ ] Documentation added or updated? If so, please share the link below
- [ ] Important decisions made with team?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "ate",
   "version": "1.0.0",
   "scripts": {
-    "start": "expo start --dev-client",
+    "start": "expo start -i",
```
Comment (byeongjae-kang) [inline package.json:5]: @hyeseonpark 
For you 😄

File: package.json [hunk 2]
```diff
@@ -72,7 +72,8 @@
     "react-native": "0.72.3",
     "react-native-gesture-handler": "~2.12.0",
     "react-native-safe-area-context": "4.6.3",
-    "react-native-screens": "~3.22.0"
+    "react-native-screens": "~3.22.0",
+    "react-native-svg": "13.9.0"
```
Comment (byeongjae-kang) [inline package.json:76]: This tool allow us to use `svg` in `react-native`

File: src/.storybook/main.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  stories: ["../components/**/*.stories.?(ts|tsx|js|jsx)"],
+  stories: ["../**/*.stories.?(ts|tsx|js|jsx)"],
```
Comment (byeongjae-kang) [inline src/.storybook/main.js:2]: We need to create `stories` files outside of components folder

File: src/components/Typography/Typography.helpers.ts [hunk 4]
```diff
@@ -1,4 +1,7 @@
-import { FontFamily, FontWeight } from "@src/components/Typography";
+import {
+  FontFamily,
+  FontWeight
+} from "@src/components/Typography/Typography.types";
```
Comment (byeongjae-kang) [inline src/components/Typography/Typography.helpers.ts:4]: This solves circular dependency warning!!

File: src/components/Typography/Typography.tsx [hunk 5]
```diff
@@ -5,7 +5,8 @@ import {
   Quicksand_700Bold,
   useFonts
 } from "@expo-google-fonts/quicksand";
-import { TypographyProps, getFontFamily } from "@src/components/Typography";
+import { getFontFamily } from "@src/components/Typography/Typography.helpers";
+import { TypographyProps } from "@src/components/Typography/Typography.types";
```
Comment (byeongjae-kang) [inline src/components/Typography/Typography.tsx:9]: This solves circular dependency warning!!

File: src/components/Typography/Typography.types.ts [hunk 6]
```diff
@@ -1,4 +1,8 @@
-import { fontColors, fontSizes, fontWeights } from "@src/components/Typography";
+import {
+  fontColors,
+  fontSizes,
+  fontWeights
+} from "@src/components/Typography/Typography.constants";
```
Comment (byeongjae-kang) [inline src/components/Typography/Typography.types.ts:5]: This solves circular dependency warning!!

File: package.json (hunk 1 above)
Comment (hyeseonpark) [reply package.json:5]: Thank you for solving! 😌

File: src/navigation/Platform/Platform.tsx
Comment (hyeseonpark) [inline src/navigation/Platform/Platform.tsx:1]: I like the way you create the Navigator! Thanks 😄

**Review by hyeseonpark — APPROVED** [review]
Thank you for your work. I'll refer to your code! 😀

**Pull request merged**